brayns/common/types.h

```diff
@@ -202,14 +192,25 @@ typedef std::shared_ptr<DeflectPlugin> DeflectPluginPtr;
 class KeyboardHandler;
 typedef std::shared_ptr<KeyboardHandler> KeyboardHandlerPtr;
 
+/** Types of renderers */
+enum class RendererType
+{
+    basic,
+    proximity,
+    simulation,
+    particle
+};
+typedef std::vector<RendererType> RendererTypes;
+typedef std::map<RendererType, RendererPtr> RendererMap;
+
 /** Define the frame buffer format */
-enum FrameBufferFormat
+enum class FrameBufferFormat
 {
-    FBF_RGBA_NONE,
-    FBF_RGBA_I8,
-    FBF_BGRA_I8,
-    FBF_RGB_I8,
-    FBF_RGBA_F32,
+    rgba_none,
```

Comment:
Do idea, that makes no sense :-( I removed it. Probably a left over from some previous experiments. Thanks.